crates/fluvio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.21.0"
+version = "0.22.0"
```

Comment:
I think that 0.21.0 is not released

Reply:
fluvio version is bump up due to change in generic signature